frontend/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <script type="module" src="./src/index.tsx"></script>
+    </head>
+    <body>
+        <div id="root"></div>
```

Comment:
Pop quiz: Why did I give this div the ID of `root`?

Reply:
(If you need a hint, search the repo for "root")

Reply:
we use it for getElementById, but I don't remember it's use.  I tried looking in the document we use, but it has a ton of 'root' references...